timeline_server.py

```
@@ -0,0 +1,150 @@
+@app.get("/v1/timeline/items/{item_id}")
+def get_item_by_id(item_id: str) -> Dict[str, Any]:
+    item = get_item(item_id)
+    if not item:
+        raise HTTPException(status_code=404, detail="Item not found")
+    return item
+
+
+@app.post("/v1/timeline/items")
+def create_item(payload: TimelineItemCreate) -> Dict[str, Any]:
+    item = add_item(payload.dict())
+    return item
```

Ruff (0.15.21)

[warning] Unnecessary assignment to `item` before `return` statement

Remove unnecessary assignment

(RET504)